### ISSUE TITLE
Graphical bug in VST3 + REAPER64 + macOS

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/Dplug#315]
VST3 only? No.
Debug-only? No.
```sh
#!/bin/sh
dplug-build -c VST
rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
sudo cp -R builds/macOS-64b-VST/*.vst /VST/
echo "r" > .lldbinit
lldb /Applications/REAPER64.app
```

**Comment by p0nce** [comment]
Fixed with 3ee17bea19243d5ad559cd2751db8bfa1fa6fed6, introduced between 29 Sep and 15 Oct for the v8 transition. All Cocoa window would be affected (hence, all macOS 64-bit plugins), update to latest tag v8.1.2 to have it fixed.
Don't use v8 before v8.1.2 for production!

**Comment by p0nce** [comment]
@0xEAB Yes, bit only if you use the new v8
